topics/variant-analysis/tutorials/sars-cov-2/tutorial.md

```diff
@@ -0,0 +1,408 @@
+---
+layout: tutorial_hands_on
+
+title: "From NCBI's Sequence Read Archive (SRA) to Galaxy: SARS-CoV-2 variant analysis"
```

Comment:
could be nice to add `sars-cov-2` as a tutorial tag

```
tags:
  - SARS-CoV-2
```

Reply:
we are using covid-19 in others tutorials, so I added this one